mocks/logger_mock.go

```diff
@@ -0,0 +1,21 @@
+package mocks
```

Comment:
Let's not call this a mock logger or put it in the mock folder, we don't want to confuse existing mocking setup. I'd rather we call this `NopLogger`, as it performs no operations. You could stick this in the same file as the interface, it's just an implementation of the interface then.